include/pmacc/Environment.hpp

```diff
@@ -490,9 +490,11 @@ namespace detail
             const int tryDeviceId = (deviceOffset + deviceNumber) % num_gpus;
 
             log<ggLog::CUDA_RT>("Trying to allocate device %1%.") % tryDeviceId;
+
 #if (PMACC_CUDA_ENABLED == 1)
             cudaDeviceProp devProp;
-            CUDA_CHECK((cuplaError_t)cudaGetDeviceProperties(&devProp, tryDeviceId));
+
+            CUDA_CHECK((cuplaError_t)ALPAKA_API_PREFIX(GetDeviceProperties)(&devProp, tryDeviceId));
```

Comment:
Should this code stay only for CUDA? (I do not know, just wondering if that is now supposed to be CUDA and HIP)